tests/Oryx.Integration.Tests/Node/NodeAngularAppTest.cs

```diff
@@ -25,8 +24,8 @@ public NodeAngularAppTest(ITestOutputHelper output, TestTempDirTestFixture fixtu
 
         // Official Node.js version that is supported by Angular CLI 6.0+ is 8.9 or greater
         [Theory]
-        [InlineData("8")]
-        [InlineData("9.4")]
+        [InlineData("8"), Trait("category", "node-8")]
+        [InlineData("9.4"), Trait("category", "node-9")]
```

Comment:
Should "node-9" be "node-9.4" here? Or the corresponding InlineData be updated?

Reply:
Thanks for catching this @daniv-msft! I've updated it and would need re-approval please :)